1209/CF20/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: SCK 2.0
+owner: Smart Citizen
```

Comment:
This needs to match the directory name of the org file.

Reply:
Just updated `owner` to match the folder name as `smartcitizen`

I hope now it's fine!